sdk/turing/experiment_config.py

```diff
@@ -0,0 +1,35 @@
+import turing.generated.models
+from typing import List, Dict
+from turing.generated.model_utils import OpenApiModel
+
+
+class ExperimentConfig:
+    def __init__(self,
+                 type: str = "nop",
+                 config: Dict = None):
+        self.type = type
+        self.config = config
+
+    @property
+    def type(self) -> str:
+        return self._type
+    
+    @type.setter
+    def type(self, type: str):
+        self._type = type
+
+    @property
+    def config(self):
+        return self._config
+
+    @config.setter
+    def config(self, config):
+        if config is not None and 'project_id' in config:
+            config['project_id'] = int(config['project_id'])
+        self._config = config
```

Comment:
This `project_id` check is a little awkward; it's there because when `project_id` gets returned as a response within the `config` attribute, its value gets parsed as a `float` by the autogenerated API classes, which when sent back as part of a request, receives a 400 Bad Request message since the API expects an `int` for the `project_id` variable.